### PR TITLE
Added CI config file.

### PR DESCRIPTION
**Pull request by Sokario**

### CODE REVIEW

**Comment by Sokario** [comment]
Errors must be fixed in another PR.

File: boards/catie/zest_core_stm32g474ve/zest_core_stm32g474ve.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+supported:
+  - pwm
+  - i2c
+  - gpio
+  - counter
+  - spi
+  - dma
+  - can
+  - rtc
+vendor: catie
```
Comment (bperraudin) [inline boards/catie/zest_core_stm32g474ve/zest_core_stm32g474ve.yaml:19]: File format is incorrect, it should be terminated by a new-line.

**Review by clementlgl — CHANGES_REQUESTED** [review]
The [github action file](https://github.com/catie-aq/cookiecutter_zephyr/blob/34-add-ci-to-templates/core/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows/board.yaml) should be added

**Comment by Sokario** [comment]
> Two remarks on format.
> 
> I also have a question: the test file for the Zest_Core_STM32L4A6RG doesn't support as many tests, and contains an `ignore_tags` section. Don't we need the same for this board?

@bperraudin The file is inspire from ST Nucleo boards like Zest_Core_STM32L4A6RG. So the differences came from the ST files.

File: boards/catie/zest_core_stm32g474ve/zest_core_stm32g474ve.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+identifier: zest_core_stm32g474ve
+name: Zest Core STM32G474VE
```
[the remark this answers is not in the document]
Comment (Sokario) [reply boards/catie/zest_core_stm32g474ve/zest_core_stm32g474ve.yaml]: Resolved in [f5f8e15](https://github.com/catie-aq/zephyr_zest-core-stm32g474ve/pull/16/commits/f5f8e156bdccabd545913a33a613e69212b42e58).

File: boards/catie/zest_core_stm32g474ve/zest_core_stm32g474ve.yaml (hunk 1 above)
Comment (Sokario) [reply boards/catie/zest_core_stm32g474ve/zest_core_stm32g474ve.yaml]: Resolved in [f5f8e15](https://github.com/catie-aq/zephyr_zest-core-stm32g474ve/pull/16/commits/f5f8e156bdccabd545913a33a613e69212b42e58).

File: .github/workflows/board.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: "Zephyr Zest_Core_STM32G474VE CI/CD"
```
Comment (bperraudin) [inline .github/workflows/board.yaml:1]:
```suggestion
name: "Test board"
```

**Review by bperraudin — APPROVED** [review]
Thanks :+1:

**Review by clementlgl — APPROVED** [review]

**Pull request merged**